be/src/vec/columns/column_string.h

```diff
@@ -32,6 +32,8 @@
 #include "vec/common/sip_hash.h"
 #include "vec/core/field.h"
 
+#include "../../util/mem_util.hpp"
```

Comment:
Maybe it's better to use an absolute path?

Reply:
+1

Reply:
`#include "util/mem_util.hpp"` also works.

Reply:
ok